app/src/ui/lib/tooltip.tsx

```diff
@@ -53,6 +53,15 @@ export interface ITooltipProps<T> {
    */
   readonly interactive?: boolean
 
+  /**
+   * Whether or not the tooltip should be dismissable via the escape key. This
+   * is generally true, but if the tooltip is communicating something important
+   * to the user, such as an input error, it should not be dismissable.
+   *
+   * Defaults to true
+   */
+  readonly dismissable?: boolean
```

Comment:
This appears to be unused? Was this supposed to be included in the `onKeyDown` conditional perhaps?

Reply:
Heh.. 🙈 well it was at some point cause I made the demo video. :P I was context switching toward the end of this PR which I should know by now is bad idea. Looks like I lost this change in the shuffle. Thank you for doing a thorough review. ❤️